symengine/functions.cpp

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Is there a specific reason to have an empty line at the beginning of the file? If not, then I would remove it.